README.md

```diff
@@ -10,14 +10,49 @@ WebdriverIO
 
 #### [Homepage](http://webdriver.io) | [Developer Guide](http://webdriver.io/guide.html) | [API Reference](http://webdriver.io/api.html) | [Contribute](http://webdriver.io/contribute.html)
 
-This library is a webdriver module for Node.js. It makes it possible to write
-super easy selenium tests in your favorite BDD/TDD test framework. It was
-originated by [Camilo Tapia's](https://github.com/camme) initial selenium project
-called WebdriverJS which was the first webdriver project on NPM.
+This library is a [webdriver](http://google-opensource.blogspot.com/2009/05/introducing-webdriver.html)
+(browser automation) module for Node.JS. It makes it possible to write
+super easy [Selenium](https://en.wikipedia.org/wiki/Selenium_(software)) tests in your favorite
+BDD/TDD test framework, that will run locally or in the cloud using Sauce Labs, Browsertack or Testingbot.
 
-Have a look at the many [examples](examples/).
+## Syantax example
 
-## How to install it
+```js
+client
+    .url('http://google.com')
+    .setValue('#q', 'webdriver')
+    .click('#btnG')
+```
+
+Notice how this is far simpler than with the original [selenium-webdriverjs](https://github.com/SeleniumHQ/selenium/wiki/WebDriverJs),
+
+```js
+driver.get('http://www.google.com');
+driver.findElement(webdriver.By.id('q')).sendKeys('webdriver');
+driver.findElement(webdriver.By.id('btnG')).click();
+```
+
+and significantly simpler than with [WD.js](https://github.com/admc/wd):
+
+```js
+browser
+  .get("http://www.google.com")
+  .elementById('q')
+  .sendKeys('webdriver')
+  .elementById('btnG')
+  .click()
+```
+
+For more details on the comparison between WebdriverIO, selenium-webdriverjs and WD.js,
+read [this discussion](https://github.com/webdriverio/webdriverio/issues/138).
+
+## History
```

Comment:
can we replace the history section with the plugin section? I think all these images are more catchy to people